src/persistent_log/fs.rs

```diff
@@ -387,7 +387,7 @@ mod test {
         }
 
         // New store with the same backing file starts with the same state.
-        let store = FsLog::new(&filename).expect("RECREATE FSLOG");
+        let store = FsLog::new(&filename).unwrap();
```

Comment:
This is in the tests.  If it fails to create the log, the test should panic.  Unwrap seems reasonable in this context.